splice/models.py

```diff
@@ -6,6 +6,16 @@
 metadata = db.metadata
 
 
+class Content(db.Model):
+    __tablename__ = "contents"
+
+    id = db.Column('id', db.Integer(), autoincrement=True, primary_key=True, info={"identity": [1, 1]})
+    name = db.Column('name', db.String(length=255), nullable=False, unique=True)
+    version = db.Column('version', db.Integer(), nullable=False)
+    created_at = db.Column('created_at', db.DateTime(), server_default=db.func.now(), nullable=False)
```

Comment:
If not, this leaves us to an attack vector:

1. CDN owner is compromised, changes original zip file as well as HTML files
2. We discover the issue, regenerate keys and re-sign
3. We sign the compromised original zip file